TypeScript.YAML-tmLanguage

```diff
@@ -361,6 +360,7 @@ repository:
     - include: '#logic-operator'
     - include: '#assignment-operator'
     - include: '#storage-keyword'
+    - include: '#function-call'
```

Comment:
Personally I don't see `#function-call` being used anywhere. Cases that I have tried:
```
foo(1);
var x = foo(1);
let x = foo<number>(1, 2);
```
Considering remove this `#function-call` if it is not necessary. Or perhaps there is a problem somewhere else.